install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -51,6 +51,13 @@ spec:
         (time()-cluster_version_operator_update_retrieval_timestamp_seconds) >= 3600 and ignoring(condition, name, reason) cluster_operator_conditions{name="version", condition="RetrievedUpdates", endpoint="metrics", reason!="NoChannel"}
       labels:
         severity: critical
+    - alert: UpdateAvailable
+      annotations:
+        message: $value {{ if eq $value 1 }}update is{{ else }}updates are{{ end }} available.  For more information refer to 'oc adm upgrade'{{ "{{ with $console_url := \"console_url\" | query }}{{ if ne (len (label \"url\" (first $console_url ) ) ) 0}} or {{ label \"url\" (first $console_url ) }}/settings/cluster/{{ end }}{{ end }}" }}.
```

Comment:
`cluster_version_available_updates > 1` would return true, not the number of alerts, right? 
In any case I don't see a benefit of mentioning number of alerts - even if its 9999 updates available admins are usually guided by maintenance windows

Reply:
Rebased onto master and reworded to be less dynamic with b397d8f3 -> ccf08821.